readthedocs/doc_builder/exceptions.py

```diff
@@ -24,7 +24,9 @@ class BuildEnvironmentError(BuildEnvironmentException):
 
     GENERIC_WITH_BUILD_ID = ugettext_noop(
         'There was a problem with Read the Docs while building your documentation. '
-        'Please report this to us with your build id ({build_id}).',
+        'Please try again later. '
+        'However, if this problem persists, '
+        'please report to us with your build id ({build_id}).',
```

Comment:
I said "please report **this** to us".

I'm not sure if that was important for the grammar, but I just wanted to mention it.

Reply:
okay.... that looks more good to me too.